rosdep/base.yaml

```diff
@@ -3184,13 +3186,20 @@ libftgl-dev:
   fedora: [ftgl-devel]
   gentoo: [media-libs/ftgl]
   ubuntu: [libftgl-dev]
+libgazebo-dev:
+  debian: [libgazebo-dev]
```

Comment:
@chapulina could you check the CI error?

> Package 'libgazebo-dev' could not be found for debian buster on amd64

If that should be working, we can relaunch the tests to see if it happens again.

Reply:
Fixed in [20ffdf5](https://github.com/ros/rosdistro/pull/32481/commits/20ffdf56ecc768ae216d3323a36f026c8484bf4d)